plonetheme/motion/profiles/uninstall/metadata.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<metadata>
```

Comment:
GS doesn't read/need metadata.xml so I suggest removing this one

Reply:
@davilima6 done with commit 6aa52ac